TACC/Core-Portal
# task/FP-627: Forbid moving/copying folders into themselves

… selected

## Overview: ##
Adds a filter to remove folders from the modal listing if those folders are selected.
## PR Status: ##


## Related Jira tickets: ##

* [FP-627](https://jira.tacc.utexas.edu/browse/FP-627)

## Summary of Changes: ##

## Testing Steps: ##
1. Select a folder and attempt to move or copy it. That folder should be hidden in the right-hand modal listing.

## UI Photos:

## Notes: ##
